<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QLuoAnAngle</class>
 <widget class="QWidget" name="QLuoAnAngle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>38</y>
     <width>691</width>
     <height>401</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="autoScroll">
    <bool>false</bool>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="return_btn">
   <property name="geometry">
    <rect>
     <x>753</x>
     <y>385</y>
     <width>48</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10;background-color: rgb(100, 100, 100);font-size:22px;color:rgb(255, 255, 0);

</string>
   </property>
   <property name="text">
    <string>返
回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>裸氨使能设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="download_btn">
   <property name="geometry">
    <rect>
     <x>753</x>
     <y>195</y>
     <width>48</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10;background-color: rgb(100, 100, 100);font-size:22px;color:rgb(255, 255, 0);

</string>
   </property>
   <property name="text">
    <string>下
载</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>753</x>
     <y>290</y>
     <width>48</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10;background-color: rgb(100, 100, 100);font-size:22px;color:rgb(255, 255, 0);

</string>
   </property>
   <property name="text">
    <string>保
存</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tips">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>440</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;
font-size:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
